frontend/server/src/DAO/QualityNominations.php

```diff
@@ -340,8 +341,12 @@ function (string $type): string {
         if (!empty($conditions)) {
             $sqlFrom .= ' WHERE ' . implode(' AND ', $conditions);
         }
+        //error_log($onlyOpen.'---2');
+        if ($onlyOpen) {
+            $sqlFrom .= " AND qn.status = 'open' ";
+        }
 
-        $sqlOrder = ' ORDER BY qn.qualitynomination_id';
+        $sqlOrder = ' ORDER BY p.alias asc, time desc';
```

Comment:
```suggestion
        $sqlOrder = ' ORDER BY p.alias ASC, qn.qualitynomination_id ASC';
```

el `qualitynomination_id` y el tiempo van en el mismo orden y es un poco más barato filtrarlo por el primero. también, el orden ascendente hace que se atiendan las nominaciones más viejas primero.

(y recuerda que todas las palabras reservadas de SQL van en mayúsculas).

Reply:
Tenemos muchas nominaciones repetidas y que se resolvieron del alguna u otra forma, pero que el duplicado sigue ahi. Si ponemos el orden ascendente primero vemos las del 2017.. Entonces preferimos ver las más nuevas primero.
Cuando tengamos una solución para los duplicados, ponemos esto en orden ASC

Reply:
vale, entonces 

```suggestion
        // TODO(...): Change to ASC once duplicates are removed.
        $sqlOrder = ' ORDER BY p.alias ASC, qn.qualitynomination_id DESC';
```